core/src/main/scala/org/apache/spark/sql/hive/SnappyStoreHiveCatalog.scala

```diff
@@ -223,6 +230,7 @@ class SnappyStoreHiveCatalog(externalCatalog: ExternalCatalog,
       metadataConf.setVar(HiveConf.ConfVars.METASTOREWAREHOUSE, warehouse)
     }
     logInfo("Default warehouse location is " + warehouse)
+    metadataConf.setVar(HiveConf.ConfVars.HADOOPFS, "file:///")
```

Comment:
See comment in https://github.com/SnappyDataInc/spark/pull/3#issuecomment-243083705

Looks to me that it will be better to change "./warehouse" to sqlConf.warehousePath and drop the Spark PR.